io/disk/fs_mark.py.data/fs_mark.yaml

```diff
@@ -0,0 +1,23 @@
+# fs_mark options:
+#  -n 10000000: The number of files to be tested per thread (more on that later)
+#  -s 400; Each file will be 400KB
+#  -L 1: Loop the testing once (fs_mark testing)
+#  -S 0: Issue no sync() or fsync() during the creation of the file system. 
+#  -D 10000: There are 10,000 subdirectories under the main root directory
+#  -d /mnt: The root directory for testing; for this particular test, we are using only 1  #   root directory
+#   -N 1000: 1,000 files are allocated per directory
+#   -t 10: Use 10 threads for building the file system
+#   -k: Keep the file system after testing
+#   -N 10,000: Allocate 10,000 files per directory
+#   SAMPLE RUN
+#   A typical run of the program would look like this:
+# ./fs_mark -d /mnt -s 10240 -n 1000
+
+setup:
+    num_files:
+        num_files: 1000 
+    size:
+        size: 10240
+    dir:
+    	dir: /mnt 
```

Comment:
Yes, Fedora 23 Kernel 4.4.9-300.fc23.x86_64, non-root user, selinux enabled. If you will require root for this test, you can use in `setUp()` something like:
~~~
if os.geteuid() != 0:
    self.error(msg)
~~~
But that does not seem to be the case. I'd suggest any dir that does not make the test to fail by default... `/var/tmp` looks like a good candidate, what do you think?